OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
+  - cfergeau
+  - praveenkumar
+  - anjannath
+  - guillaumerose
+  - gbraad
```

Comment:
:+1:

Reply:
@guillaumerose I think you also need reviewers details to this file.

Reply:
I took the example from https://github.com/tektoncd/cli/blob/master/OWNERS. It seems to work fine for them.

Reply:
@guillaumerose alright. looks like the bot will not able to assign the reviewer if it is not there but that's OK since we don't want the bot to be that intelligent.